future.tex

```diff
@@ -108,6 +33,38 @@ \subsection{Summary}
 	}
 \end{table}
 
+The next table lists different nations in the EU and their nuclear program plans.
+The attitude is divided into five, from very negative to very positive, where 
```

Comment:
The attitude of what? Five what? (A trick for noticing when a sentence is unclear is to read the sentence aloud after you write it.) Perhaps a sentence like this would be more clear if it were `For each EU nation, the growth trajectory is categorized from "Aggressive Growth" to "Aggressive Shutdown". Aggressive growth is characterized by a rigorous expansion of nuclear power while Aggressive shutdown is characterized as a transition to rapidly de-nuclearize that nation's electric grid.`


Also, I'd like to avoid "attitude" because it's fuzzy (just because they're shutting down reactors doesn't necessarily mean they have a negative attitude toward nuclear as a nation. Maybe it's just economics). I wonder if we would do better to call it "Growth Trajectory" or similar and the file labels could be something more expressive:

- Aggressive Growth
- Modest Growth
- Maintenance
- Modest Reduction
- Aggressive Reduction